room_capacity/Code.js

```diff
@@ -0,0 +1,10 @@
+function ROOM_TYPE(numParticipants) {
+    if (numParticipants <= 25) {
+      return '🎈 Medium room: 25max';
+    }
+    if (numParticipants <= 40) {
+      return '🎉 Large room: 40max';
+    }
+    return '💥 Extra-large room: 200max'
+  }
+  
```

Comment:
I wander why Github complains about the missing newline at the end if it looks like it's there. Maybe make sure it's actually a *blank* line (no whitespaces or tabs)?